src/HashedExpression/Internal/Normalize.hs

```diff
@@ -527,6 +590,8 @@ pullOutPiecewiseRules exp@(mp, n)
     one = const_ shape 1
     zero = const_ shape 0
 
+-- FIXME: not sure about this one either
+
```

Comment:
TwiceReFT node is taking real part of fourier transform twice. (Mathematically: 
(xRe . ft) . (xRe . ft)), same as TwiceImFT.

We had them because taking ReFT (xRe . ft) is O(nlogn), but TwiceReFT is O(n).

This rule simplify even further, TwiceReFT(x) + TwiceImFT(x) is just (size(x) / 2) *. x.